test/realmodel/README.md

```diff
@@ -0,0 +1,6 @@
+# Generate realmodel testcase tool 
+
+1. Download and unzip [Squeezenet](https://s3.amazonaws.com/onnx-model-zoo/squeezenet/squeezenet1.1/squeezenet1.1.tar.gz) to `tool/generate_realmodel/model/`
```

Comment:
@zhaoming0 Can we have **config.json** file to configure the path of real model, and implement scripts to do step 1~3?